resources/views/users/includes/membershipform_include.blade.php

```diff
@@ -44,8 +14,12 @@
 </p>
 
 <p>
-    The undersigned is aware of the Bylaws (NL: Statuten) and Rules & Regulations (NL: Huishoudelijk Regelement) of the
-    association and promises to follow them.
+    The undersigned is aware of the
+    <a href="https://wiki.proto.utwente.nl/_media/proto/statues_v3_eng_censor.pdf" target="_blank">Bylaws</a>
```

Comment:
I'd link to the wiki page with the most up to date document. Chances are we'll forget to update these links in the future.